web/src/lib/components/Auction.svelte

```diff
@@ -101,7 +101,7 @@
                     {/if}
                 </div>
                 <div class="lg:w-1/3">
-                    <p class="mt-4">{auction.description}</p>
+                    <p class=css"mt-4">{auction.description}</p>
```

Comment:
typo?

Reply:
Yes, however I caught this typo and it was fixed in [af9374a](https://github.com/PlebeianTech/plebeian-market/pull/6/commits/af9374ab038ccad545d6d1e332ba1bfc99d950fa) -- probably should have amended this commit instead